### PR TITLE
Document the purpose of the chairman tests

### PR DESCRIPTION
**Pull request by newhoggy**
This is to address this ticket: https://github.com/input-output-hk/cardano-node/issues/671

### CODE REVIEW

File: doc/getting-started/running-chairman-tests.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+The chairman test will run some testnets and perform some basic tests to ensure they
+are working.
+
+Those basic tests may differ depending on the testnet that is being tested, but may
+include for example:
+
+Asserting that each node:
+
+* Opens the correct port
+* Create a socket file (or OS-specific equivalent) at the expected location
```
Comment (erikd) [inline doc/getting-started/running-chairman-tests.md:34]: "Creates" ?

**Review by erikd — APPROVED** [review]
One minor type needs fixing but otherwise LGTM!

File: doc/getting-started/running-chairman-tests.md (hunk 1 above)
Comment (newhoggy) [reply doc/getting-started/running-chairman-tests.md]: Fixed.  Thanks!

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**